packages/bsky/src/services/indexing/index.ts

```diff
@@ -54,6 +56,7 @@ export class IndexingService {
       profile: Profile.makePlugin(this.db, backgroundQueue),
       list: List.makePlugin(this.db, backgroundQueue),
       listItem: ListItem.makePlugin(this.db, backgroundQueue),
+      block: Block.makePlugin(this.db, backgroundQueue),
```

Comment:
I think we should remove records from the `actor_block` table during `unindexActor()` too.

Reply:
oh yup totally. good call 👍